infra/README.md

```diff
@@ -23,6 +23,9 @@ Commands:
   help [<command>...]
     Show help.
 
+  gke show deploymentvars
+    gke show deploymentvars -a service-account.json
```

Comment:
Maybe `infra gke info` will be a better command for this :) what do you think? also would be nice if we could do this without having to pass the auth file. This should become easier after https://github.com/prometheus/test-infra/pull/413/

Reply:
Done